backend/app/build.gradle.kts

```diff
@@ -124,3 +131,14 @@ tasks.withType<Detekt>().configureEach {
     md.required.set(true)
   }
 }
+
+allprojects {
+  apply(plugin = "org.owasp.dependencycheck")
```

Comment:
:information_source: this adds a Gradle plugin that pulls (and caches) the cves, which are then used to scan the code against.